experimental/sdg-tracker/src/state/index.ts

```diff
@@ -52,6 +52,42 @@ const MenuImageIcon = styled.img`
 `;
 
 const REGION_PLACE_TYPES = ["UNGeoRegion", "ContinentalUnion", "Continent"];
+
+const EARTH_DCID = "Earth";
+
+const EARTH_COUNTRIES = [
+  "country/AUS",
+  "country/USA",
+  "country/MEX",
+  "country/BRA",
+  "country/COL",
+  "country/IND",
+  "country/CHN",
+  "country/RUS",
+  "country/NGA",
+  "country/ETH",
+  "country/SOM",
+  "country/KEN",
+  "country/FRA",
+  "country/UKR",
+  "country/DEU",
+  "country/JPN",
+  "country/IDN",
+  "country/PHL",
+  "country/SAU",
+  "country/IRN",
+  "country/KAZ",
+];
+
+const CONTINENTS = new Set([
+  "africa",
+  "antartica",
+  "asia",
+  "europe",
+  "northamerica",
+  "oceania",
+]);
+
```

Comment:
Can you move these constants to https://github.com/datacommonsorg/website/blob/62d97f294f5d1ce229443bf70eb40826caf1fb82/experimental/sdg-tracker/src/utils/constants.ts

(also can you use the existing "Earth" constant here https://github.com/datacommonsorg/website/blob/62d97f294f5d1ce229443bf70eb40826caf1fb82/experimental/sdg-tracker/src/utils/constants.ts#L22)

Reply:
Done.